slave/process-drupal-elixir/bin/process-drupal_elixir.sh

```diff
@@ -36,4 +38,10 @@ function process {
 			log_msg I_NOT_CHANGED
 		fi
 	done
+
+	#there are some duplicities, need to end with error
+  if [ -s "$FILE_USERS_DUPLICITIES" ]; then
+		cat "$E_DUPLICITIES" >&2;
+		log_msg E_DUPLICITIES
```

Comment:
Shouldn't one of them print content of ``users-duplicities.csv`` ? or do you just want to end up with the error code alone?

Reply:
You are right, there is wrong variable. Fixed.